.gitignore

```diff
@@ -1,4 +1,43 @@
-/vendor/*
+# CakePHP specific files #
+##########################
 /config/app.php
-/tmp/*
+/config/.env # josegonzalez/php-dotenv
 /logs/*
+/tmp/*
+/tmp/bake/*
```

Comment:
I think the tools are really the thing we should discuss.
I don't think adding `/tmp/*` is good practise. The `empty` files should stay and without those extra directives, they would not.

p.s.: Personally I am using `.keep` however as that is much better than `empty` (says more about why the file exists, is hidden by default, etc)